### PR TITLE
Fix hasBeenWarned condition.

### PR DESCRIPTION
**Pull request by yavorsky**
Remove check for static `hasBeenWarned` that always `false`.

### CODE REVIEW

File: src/normalize-options.js [hunk 1]
```diff
@@ -71,7 +69,7 @@ export const validateModulesOption = (modulesOpt = "commonjs") => {
 
 export default function normalizeOptions(opts) {
   // TODO: remove whitelist in favor of include in next major
-  if (opts.whitelist && !hasBeenWarned) {
+  if (opts.whitelist) {
     console.warn(
```
Comment (hzoo) [inline src/normalize-options.js]: then we should add hasBeenWarned = true in the if block? otherwise it will warn multiple times?

Comment (yavorsky) [inline src/normalize-options.js]: @hzoo yes! I thought it could be called once, but actually not. Updated!

**Comment by existentialism** [comment]
Should we add a test for this?

**Comment by hzoo** [comment]
oh yeah, always

**Comment by hzoo** [comment]
maybe bot should warn - if you change src and no test files changed 😼

**Comment by yavorsky** [comment]
Maybe pre-commit hook? You can't commit unless added some tests 🙂

I'll cover it after lunch (UTC +2).

**Comment by hzoo** [comment]
Nah since I don't want new contributors to not make the pr if they don't have it, can tell them after?

**Comment by yavorsky** [comment]
I was joking :) But the bot that shows changes coverage - great idea!

**Comment by hzoo** [comment]
And we do have codecov set up on babel already

**Pull request merged**